Description: Adjust the type of the guava dependency in the guava-testlib module
Author: Emmanuel Bourg <ebourg@apache.org>
Forwarded: not-needed
--- a/guava-testlib/pom.xml
+++ b/guava-testlib/pom.xml
@@ -34,6 +34,7 @@
       <groupId>${project.groupId}</groupId>
       <artifactId>guava</artifactId>
       <version>${project.version}</version>
+      <type>bundle</type>
     </dependency>
     <dependency>
       <groupId>junit</groupId>
